### PR TITLE
Design

### PR DESCRIPTION
**Pull request by Tymac15**
Added the wireframe diagrams for the following pages.

### CODE REVIEW

**Review by Bandisa — CHANGES_REQUESTED** [review]
# ReadMe
Please add a `ReadMe.txt'
# File structure
Please followed the prosed file structure as per last meeting.
image

in you case it would be `design/front-end/UI/{pages, components, animations...etc}

**Review by u21598267 — APPROVED** [review]
Looks good.

File: Design/UI designs/Menu page.jpg
Comment (Bandisa) [inline Design/UI designs/Menu page.jpg:1]: # Feature request
1. A horizontal bar or highlight that will indicate currrent page on this nav bar if a user is currently on one. eg, if a user is currently editing a time table, `Edit timetable` Should be underlined, highlighted or some other fitting means as you see fit` to indicate that current page.

File: Design/UI designs/Schedule.jpg
Comment (Bandisa) [inline Design/UI designs/Schedule.jpg:1]: This looks great.

# Feature request
- A mechanism to indicate current on going event in real time.
- eg, if it is monday and the time is 08:37, IMY310 should be scaled up, highlighted, saturation increased of,  a horizontal bar extending from the 08:37 time point, and/or vertically from the middle of Monday axis.

File: Design/UI designs/Menu page.jpg
Comment (Tymac15) [reply Design/UI designs/Menu page.jpg:1]: I see what you're saying but the menu page ('nav bar") doesn't remain when you choose to edit the timetable, you go to a whole new page which can be exited if you take the "go back action" which will take you back to the menu. same for the other options in the menu

File: Design/UI designs/Schedule.jpg
Comment (Tymac15) [reply Design/UI designs/Schedule.jpg:1]: As advised by Shashin, these are the basic designs and we will implement features like that once everything else has been implemented well.

File: Design/UI designs/Menu page.jpg
Comment (Bandisa) [reply Design/UI designs/Menu page.jpg:1]: Okay great. That makes sense. 
I love it 💪🏾.

**Comment by ShashinGounden** [comment]
Discussed and showcased the designs in our meeting yesterday and it looks good as a starting point for our front-end implementation. We will take these designs as a starting point for our front-end implementation in ReactNative.

**Pull request closed without merging**